<!--
  Arguments:
  lat   : Home latitude
  lon   : Home longitude
  alt   : Home altitude

  freq  : Update frequency of the PX4_msgs SensorGps message
  n_off : Offset angle from geographical north to the vivo tracker calibrated y-axis
-->

<launch>
  <arg name="lat" default="61.50361"/>
  <arg name="lon" default="23.77500"/>
  <arg name="alt" default="110.0"/>
  <arg name="freq" default="10"/>
  <arg name="n_off" default="0.0"/>

  <node name="indoor_pos" pkg="indoor_pos" exec="indoor_pos" namespace='$(env DRONE_DEVICE_ID)'>
    <param name="home_lat" value="$(var lat)"/>
    <param name="home_lon" value="$(var lon)"/>
    <param name="home_alt" value="$(var alt)"/>
    <param name="frequency" value="$(var freq)"/>
    <param name="north_offset" value="$(var n_off)"/>
  </node>
</launch>

